libs/dh/reports/feature-measurement-reports/src/components/measurement-reports.component.ts

```diff
@@ -16,111 +16,94 @@
  */
 //#endregion
-import { Component, computed, inject } from '@angular/core';
+import { Component, computed } from '@angular/core';
 import { TranslocoDirective } from '@jsverse/transloco';
-import { toSignal } from '@angular/core/rxjs-interop';
 
-import { VaterUtilityDirective } from '@energinet-datahub/watt/vater';
-import { WattTableColumnDef, WATT_TABLE, WattTableDataSource } from '@energinet-datahub/watt/table';
-import { WattDataTableComponent } from '@energinet-datahub/watt/data';
-import { WattDatePipe } from '@energinet-datahub/watt/date';
+import {
+  VaterFlexComponent,
+  VaterSpacerComponent,
+  VaterStackComponent,
+  VaterUtilityDirective,
+} from '@energinet-datahub/watt/vater';
+import { WATT_CARD } from '@energinet-datahub/watt/card';
+import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
+import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
 
-import { PermissionService } from '@energinet-datahub/dh/shared/feature-authorization';
+import { query } from '@energinet-datahub/dh/shared/util-apollo';
+import { GetMeasurementsReportsDocument } from '@energinet-datahub/dh/shared/domain/graphql';
 
-type DhMeasurementReport = {
-  id: string;
-  startedAt: Date;
-  actorName: string;
-  meteringPoints: string[];
-  gridAreas: string[];
-  statusType: string;
-};
+import { DhNewReportRequest } from './new-report-request.component';
+import { DhOverview } from './overview/overview.component';
 
 @Component({
   selector: 'dh-measurement-reports',
   imports: [
     TranslocoDirective,
+    WATT_CARD,
 
-    WATT_TABLE,
-    WattDataTableComponent,
+    VaterStackComponent,
+    VaterFlexComponent,
     VaterUtilityDirective,
-    WattDatePipe,
+    VaterSpacerComponent,
+    WattEmptyStateComponent,
+    WattSpinnerComponent,
+    DhNewReportRequest,
+    DhOverview,
   ],
-  template: `
-    <watt-data-table
-      *transloco="let t; read: 'reports.measurementReports'"
-      vater
-      inset="ml"
-      [enableSearch]="false"
-    >
-      <h3>{{ t('title') }}</h3>
-
-      <watt-table
-        [dataSource]="dataSource"
-        [columns]="columns"
-        [displayedColumns]="displayedColumns()"
-      >
-        <ng-container
-          *wattTableCell="columns['startedAt']; header: t('columns.startedAt'); let entry"
-        >
-          {{ entry.startedAt | wattDate: 'long' }}
-        </ng-container>
+  styles: `
+    :host {
+      display: block;
+    }
 
-        <ng-container
-          *wattTableCell="columns['actorName']; header: t('columns.actorName'); let entry"
-        >
-          {{ entry.actorName }}
-        </ng-container>
+    h3 {
+      margin: 0;
+    }
+  `,
+  template: `
+    <watt-card vater inset="ml" *transloco="let t; read: 'reports.measurementReports'">
+      @if (isLoading()) {
+        <vater-stack fill="vertical" justify="center">
+          <watt-spinner />
+        </vater-stack>
+      } @else {
+        @if (totalCount() === 0) {
```

Comment:
Suggestion: Maybe we could extend dhresult with a the possibility to render a action component if it has an empty state?